Source/Core/VideoCommon/PixelShaderManager.cpp

```diff
@@ -447,17 +447,17 @@ void PixelShaderManager::SetGenModeChanged()
   dirty = true;
 }
 
-void PixelShaderManager::SetZControlChanged()
+void PixelShaderManager::SetZModeControl()
```

Comment:
Was this function just not getting called before or did it get missed when adding ubershaders?